copyright

```diff
@@ -260,8 +260,10 @@ Files: images/outfit/tactical?scanner*
 Copyright: Zachary Siple
 License: CC-BY-SA-4.0
 
-Files: images/outfit/korath?rifle*
-Copyright: Evan Fluharty (Evanfluharty@gmai.com)
+Files:
+images/outfit/korath?rifle*
```

Comment:
Under `Files:` the lines should be indented (see the other entries)

Reply:
okay